hotfixes/4.0.1.cfg

```diff
@@ -25,6 +26,7 @@ Products.PloneHotfix20131210 = 1.0
 Products.PloneHotfix20150910 = 1.1
 Products.PloneHotfix20151208 = 1.0
 Products.PloneHotfix20160419 = 1.0
+Products.PloneHotfix20160830 = 1.1
```

Comment:
@jone I think I should - I deployed `1.1`. I did so by using manual additions to the respective deployment buildouts though, not by extending from this branch. So go ahead and update, I'll re-deploy on my end.